vnext/stubs/glog/logging.h

```diff
@@ -69,6 +69,7 @@ inline std::ostream &GetNullLog() noexcept {
 }
 
 #define LOG(b) GlogStub::GetNullLog()
```

Comment:
@acoates-ms we use `LOG(FATAL)` like `std::terminate()` in places, which probably should kill the app...